taxi/admin.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi.models import Driver, Car, Manufacturer
+
+
+@admin.register(Driver)
+class DriverAdmin(UserAdmin):
+    list_display = UserAdmin.list_display + ("license_number",)
+    fieldsets = UserAdmin.fieldsets + ((
+        "Additional info", {"fields": ("license_number",)}
+    ),)
+    add_fieldsets = UserAdmin.add_fieldsets + ((
+        "Additional info", {"fields": ("license_number",)}
+    ),)
+
+
+@admin.register(Car)
+class CarAdmin(admin.ModelAdmin):
+    search_fields = ("model",)
+    list_filter = ("manufacturer",)
+
+
+admin.site.register(Manufacturer)
```

Comment:
Register all models either with the use of the `register` decorator or `site.register`, one approach should be picked up for a project.